plugins/search-backend-module-elasticsearch/src/engines/ElasticSearchSearchEngine.ts

```diff
@@ -145,8 +151,8 @@ export class ElasticSearchSearchEngine implements SearchEngine {
       aliasPostfix = `search`,
       indexPrefix = ``,
     } = options;
-
-    const clientOptions = await createElasticSearchClientOptions(
+    this.credentialProvider = DefaultAwsCredentialsManager.fromConfig(config);
```

Comment:
I'm not *in love* with this implementation but couldn't think of a better way to incorporate the `DefaultAwsCredentialsManager` which I think is the preferred approach for specifying/configuring AWS creds.